docs/joss_paper/paper.md

```diff
@@ -84,10 +84,10 @@ Image fusion is a fundamental task in medical image analysis and computer-assist
 A submission to the MICCAI Educational Challenge has utilised the DeepReg code and demos to explore the link between classical algorithms and deep-learning-based methods [@brown2020introduction], while a recently published research work investigated temporal changes in prostate cancer imaging, by using a longitudinal registration adapted from the DeepReg code [@yang2020longitudinal].
 
 # Statement of need
-Currently, popular packages focusing on deep learning methods for medical imaging, such as NiftyNet [@gibson2018niftynet] and MONAI (https://monai.io/), do not support image registration. The existing open-sourced registration projects either implement specific published algorithms, such as the VoxelMorph [@balakrishnan2019voxelmorph], or focus on classical methods, such as NiftiReg [@modat2010fast], SimpleElastix [@marstal2016simpleelastix] and AirLab [@sandkuhler2018airlab]. Therefore an open-sourced project focusing on image registration with deep learning is much needed for general research and education purposes.
+Currently, popular packages focusing on deep learning methods for medical imaging, such as NiftyNet [@gibson2018niftynet] and MONAI (https://monai.io/), do not support image registration. The existing open-sourced registration projects either implement specific published algorithms without extensive testing, such as the VoxelMorph [@balakrishnan2019voxelmorph], or focus only on classical methods, such as NiftiReg [@modat2010fast], SimpleElastix [@marstal2016simpleelastix] and AirLab [@sandkuhler2018airlab]. Therefore an open-sourced project focusing on image registration with deep learning is much needed for general research and education purposes.
```

Comment:
i'd say "without automated testing"
no need for ""only

Reply:
done